reana_workflow_controller/k8s.py

```diff
@@ -179,12 +179,28 @@ def instantiate_k8s_objects(kubernetes_objects, namespace):
         current_k8s_extensions_v1beta1.create_namespaced_ingress
     }
     try:
-        for obj in kubernetes_objects.items():
+        parent_k8s_object_references = None
+        for index, obj in enumerate(kubernetes_objects.items()):
             kind = obj[0]
             k8s_object = obj[1]
-            instantiate_k8s_object[kind](namespace, k8s_object)
+            if index == 0:
```

Comment:
Shall we remove this `if` by picking the first `kubernetes_objects.items()` element directly and then iterating over the rest (if they exist) assingning them the `owner_references`?

Reply:
I was thinking about that, but IMO `instantiate_chained_k8s_objects` gets much bigger and more complicated, because `kubernetes_objects` is a dict and it is impossible to loop starting from 2nd item `kubernetes_objects.items()[1:]`, I would need to do some conversions.